bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```diff
@@ -271,6 +271,9 @@ metadata:
       ]
     capabilities: Seamless Upgrades
     categories: Streaming & Messaging
+    containerImage: quay.io/arkmq-org/activemq-artemis-operator:2.0.2
+    createdAt: 2025-04-03T15:42:02Z
```

Comment:
Is the PR check failing because the `createAt` timestamp in `bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml` doesn't match the one in `config/manifests/bases/activemq-artemis-operator.clusterserviceversion.yaml`?
https://github.com/arkmq-org/activemq-artemis-operator/actions/runs/14247174652/job/39930972140?pr=1126